.github/workflows/docker.yml

```diff
@@ -0,0 +1,34 @@
+name: Docker
```

Comment:
can't we just use goreleaser?

Reply:
If it is possible to publish releases to github with github actions, my vote is to retire goreleaser and switch to github actions for publishing to docker hub and github.

Reply:
goreleaser is still necessary for cross compilation build+changelog generation.

@sonmezonur mentioned that goreleaser can't publish multiarch docker releases, so we decided to go with docker/build-push-action **and** goreleaser-action.